OCIContainer/CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ All notable changes to this RDK Service will be documented in this file.
 ## [1.0.0] - 2022-05-11
 ### Added
 - Add CHANGELOG
-
+- Added unit tests for OCIContainer
```

Comment:
you need to update the revision (1.0.0->1.0.1) number in https://github.com/rdkcentral/rdkservices/blob/main/OCIContainer/OCIContainer.cpp#L10 and have a separate entry for 1.0.1 in CHANGELOG.md file